scripts/installDependencies.sh

```diff
@@ -1,9 +1,11 @@
 # Install nodejs dependencies
-sudo apt update
+apt update
 echo "Current DIR: $PWD"
-# Script to install Node.js using NVM on Ubuntu without sudo
+# Script to install Node.js using NVM on Ubuntu
+
+# Read node version from .nvmrc file
+node_version=$(cat .nvmrc | tr -d "\n")
 
-node_version=v20
 # Function to install NVM
 install_nvm() {
     curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
```

Comment:
Optional, but we can already bump to the latest available nvm. We can make this to be dynamic in the future.
```suggestion
    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.0/install.sh | bash
```

Reply:
Done.